src/marks/mark.py

```diff
@@ -84,9 +85,7 @@ class CustomizableTerm:
     rule=(
         "<user.cursorless_decorated_symbol> | "
         "{user.cursorless_special_mark} |"
-        # Because of problems with performance we have to have a simple version for now
-        # "<user.cursorless_line_number>" # row, up, down
-        "<user.cursorless_line_number_simple>"  # up, down
+        "<user.cursorless_line_number>"  # row, up, down
```

Comment:
```suggestion
        "<user.cursorless_line_number>"  # row mod 100, up, down
```

Reply:
The comment here is made more as a reference to the spoken phrase and not the actual implementation

Reply:
Hmm ok.  Just wanted to have some indication somewhere that we're currently only supporting modulo.  Maybe the following?

```suggestion
        "<user.cursorless_line_number>"  # row (ie absolute mod 100), up, down
```

Reply:
Works for me